src/main.rs

```diff
@@ -1,3 +1,10 @@
-fn main() {
+mod db;
+
+#[tokio::main]
+async fn main() {
+    if let Err(e) = db::run_db_operations().await {
```

Comment:
Abstract that out to a variable; conciseness is good but this harms readability.